.tekton/discovery-ui-pull-request.yaml

```diff
@@ -505,21 +519,22 @@ spec:
         operator: in
         values:
         - "false"
-      workspaces:
-      - name: workspace
-        workspace: workspace
     - name: sast-unicode-check
       params:
       - name: image-url
         value: $(tasks.build-image-index.results.IMAGE_URL)
+      - name: SOURCE_ARTIFACT
+        value: $(tasks.prefetch-dependencies.results.SOURCE_ARTIFACT)
+      - name: CACHI2_ARTIFACT
+        value: $(tasks.prefetch-dependencies.results.CACHI2_ARTIFACT)
       runAfter:
       - build-image-index
       taskRef:
         params:
         - name: name
-          value: sast-unicode-check
+          value: sast-shell-check-oci-ta
```

Comment:
Just based on the pattern I've seen so far, should this be `sast-unicode-check-oci-ta`, or does `sast-shell-check-oci-ta` replace _both_ `sast-shell-check` and `sast-unicode-check` ?

Reply:
TBH, I've no idea. this particular change came from konflux, so I'm assuming this is the correct task.

Reply:
Actually, I'm sure it is right. if the right tasks are not being used we should be seeing EC failures.

Reply:
🤞 I will trust the system.